shared-data/default-theme/html/jsapi/search/ui.js

```diff
@@ -79,6 +75,14 @@ Mailpile.results_list = function() {
 };
 
 
+Mailpile.display_keybindings = function(elem) {
+  Mailpile.API.with_template("modal-display-keybindings", function(modal) {
```

Comment:
Would it make sense to check here if there is already a modal active, and do nothing in that case? We don't want to clobber an ongoing action just because the user hit ? by accident. Since our current UI can only have one modal active at a time, this becomes a problem.